### ISSUE TITLE
New release

### ISSUE THREAD

**Issue by mnaser** [1Password/connect-helm-charts#19]
Hi folks :)

Could we have a new tagged release in order to capture the changes involving .Release.Namespace?

Thank you!

**Comment by mnaser** [comment]
For those who would like a workaround running 1.0.1 or before:

```
helm -n 1password-connect install connect 1password/connect --set-file connect.credentials=1password-credentials.json --set namespace=1password-connect
```

**Comment by mnaser** [comment]
Would also be nice if we landed #20 before hand, but that may be because of my personal selfish reasons to avoid running a fork :)

**Comment by jpcoenen** [comment]
Happy to announce we just released `v1.1.0`! :tada: It also includes #20.
